src/Streamly/Data/Serialize/Instances/ByteString.hs

```diff
@@ -0,0 +1,84 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE UnboxedTuples #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+module Streamly.Data.Serialize.Instances.ByteString () where
+
+import Streamly.Internal.Data.Serialize (Serialize(..))
+import Data.ByteString.Internal as Strict
+import Data.ByteString.Lazy as Lazy
+import qualified Streamly.Internal.Data.Unbox as Unbox
+import GHC.Exts
+import Foreign.ForeignPtr (withForeignPtr)
+import Foreign.Ptr (plusPtr)
+import GHC.Base (IO(..))
+import Control.Monad (foldM_)
```

Comment:
Keep the explicit imports first, qualified imports next, and the rest after that.
Import qualified where possible.